zstacklib/zstacklib/utils/resource_control.py

```
@@ -0,0 +1,1227 @@
+            '--property=ActiveState', '--property=ControlGroup',
+            '--property=MainPID'
+        ], self.SYSTEMD_QUERY_TIMEOUT)
+        result = {}
+        for line in output.splitlines():
+            if '=' in line:
+                key, value = line.split('=', 1)
+                result[key] = value
+        return result
+
+    def _systemctl(self, arguments, timeout):
+        process = subprocess.Popen(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)